### DISCUSSION TITLE
Transparent background?

### DISCUSSION THREAD

**Discussion by iacobson** [replit/kaboom#549] [Q&A] [answered]
I'm trying to integrate a canvas with Kaboom with an existing web project, and I need the kaboom canvas to be transparent.
I tried things like 

```js
kaboom({
  background: [0, 0, 0, 0]
})
```
but that renders just a black background.
If I completely remove the background key, I get the checkers pattern like this:

<img width="1538" alt="Screen Shot 2022-02-22 at 21 11 32" src="https://user-images.githubusercontent.com/8473345/155216180-d1816763-d834-4fed-90eb-deb5756b2f70.png">

**Comment by slmjkdbtl** [comment, marked as answer]
Just added support for transparent background, your code should work in v2000.2.9

**Reply by iacobson** [reply]
thanks!
